bitcoin/rpc.py

```diff
@@ -380,6 +380,12 @@ def dumpprivkey(self, addr):
 
         return CBitcoinSecret(r)
 
+    def getaddressinfo(self, addr):
+        """Return information about the given bitcoin address. Some information requires the address to be in the wallet."""
+        r = self._call('getaddressinfo', str(addr))
+        r['address'] = CBitcoinAddress(r['address'])
+        return r
```

Comment:
Just returning the raw RPC output isn't enough. You also should convert all the fields to `python-bitcoinlib` types where possible, like other RPC calls do.